src/repositories/__tests__/request-repository.test.ts

```diff
@@ -194,6 +126,131 @@ describe('request repository test', () => {
     expect(received).toEqual(expected)
   })
 
+  test('findAndMarkAsProcessing: process requests oldest to newest', async () => {
+    const requests = await Promise.all(
+      [...Array(2).keys()].map((_, i) =>
+        generateRequests(
+          {
+            status: RequestStatus.READY,
+            createdAt: new Date(Date.now() + MS_IN_HOUR * i),
+          },
+          1,
+          false
+        )
+      )
+    ).then((arr) => arr.flat())
+
+    await requestRepository.createRequests(requests)
+    const loadedRequests = await requestRepository.findAndMarkAsProcessing()
+
+    expect(loadedRequests.length).toEqual(2)
+    expect(loadedRequests[0].createdAt.getTime()).toBeLessThan(
+      loadedRequests[1].createdAt.getTime()
+    )
+    expect(loadedRequests[0].cid).toEqual(requests[0].cid)
+    expect(loadedRequests[1].cid).toEqual(requests[1].cid)
+  })
+
+  describe('findRequestsToGarbageCollect', () => {
+    test('Finds requests older than a month', async () => {
+      const threeMonthsAgo = new Date(Date.now() - MS_IN_MONTH * 3)
```

Comment:
I feel like these garbage collection tests are now much longer and harder to identify the core parts that matter.  Could you put back a helper function (or functions) for creating the requests to make it easier to parse what's happening here to a reader?

Considering the code block from berfore:
```
      generateCompletedRequest(false, false),
      generateCompletedRequest(true, false),
      generateCompletedRequest(false, true),
      generateCompletedRequest(true, true),
```

The nice thing about it was that it's pretty clear that there are 2 possible things we are changing about the requests and we are exhausting the 4 possible combinations of those 2 options.
The downside is that it's not immediately obvious to the reader what the 2 boolean arguments represent, but that's pretty easy to tell by looking at the definition of `generateCompletedRequest`.  Could also consider making the argument an object so the options are named at the call site and it's clearer to readers of just the calling code. E.G.: `generateCompletedRequest({expired: false, failed: false})`

Reply:
yup will do!